### ISSUE TITLE
MLE loss becomes zero immediately after first step

### ISSUE THREAD

**Issue by skeshaw** [rohithreddy024/Text-Summarizer-Pytorch#7]
I was trying to run the regular training process with MLE loss on CNN/DM dataset, before attempting any modifications. However, while the code in itself is running fine, the loss becomes zero immediately after the first step, whatever the batch size.
![pytorch_seq2seq_loss](https://user-images.githubusercontent.com/15869426/56573678-43bbbc00-65df-11e9-8779-bc8fb85cb48f.PNG)
I was wondering if you would be able to tell why such a situation might occur.
Thanks in advance.

**Comment by skeshaw** [comment]
I want to add one more thing. While trying to run MLE+RL or RL training, the only command line output I get is this:
`Loaded model at data/saved_models/0005000.tar`
After this, there is no training and the process just exits without any error.
I think the aforementioned issue and this may be somewhat related. Your help will be very useful here.

**Comment by rohithreddy024** [comment]
Don't use `batcher.py` for parsing CNN/DM data files as its written for Gigaword dataset and I addressed this in one of the closed issues. I think this might be the problem. For your second query, program exits without error if `iter` variable reaches `max_iterations` in `data_util/config.py`. So check for it.

**Comment by skeshaw** [comment]
Yes, I figured that shortly after opening this issue. Anyway, I got access to the original files and am able to run the code fine now. Thanks.
